docs/2.0. APIs.md

```diff
@@ -41,33 +43,34 @@ Where protocol and versioning data is not available, clients MAY assume a v1.0 A
 
 All public APIs are versioned as follows:
 
-* Requesting the API base resource (such as http(s)://&lt;ip address or hostname&gt;:&lt;port&gt;/x-nmos/query/) will provide a list containing the versions of the API present on the node.
-* A versioned API response must include only resources which match the schema for that API version.
-* Data which is held for mismatched minor API versions may be returned if it can be conformed to the correct schema (see example below). Data must never be conformed between major API versions.
+- Requesting the API base resource (such as http(s)://&lt;ip address or hostname&gt;:&lt;port&gt;/x-nmos/query/) will provide a list containing the versions of the API present on the node.
+- A versioned API response MUST include only resources which match the schema for that API version.
+- Data which is held for mismatched minor API versions MAY be returned if it can be conformed to the correct schema (see example below). Data MUST NOT be conformed between major API versions.
 
 **Example**
 
-A v1.1 API response may include:
-* v1.1 data without modification.
-* Data conforming to schemas less than v2.0 but greater than v1.1, with any non-v1.1 keys removed.
-* Query API only: Data confirming to schemas less than v1.1 but greater than or equal to v1.0 if a suitable downgrade parameter is specified (see [Query Parameters](2.5.%20APIs%20-%20Query%20Parameters.md)).
+A v1.1 API response can include:
+
+- v1.1 data without modification.
+- Data conforming to schemas less than v2.0 but greater than v1.1, with any non-v1.1 keys removed.
+- Query API only: Data confirming to schemas less than v1.1 but greater than or equal to v1.0 if a suitable downgrade parameter is specified (see [Query Parameters](2.5.%20APIs%20-%20Query%20Parameters.md)).
 
 ### Common API Base Resource
 
-```
+```json
 [
   "v1.0/",
   "v2.0/",
   "v3.0/"
 ]
 ```
 
-* Appending /v1.0/ to the API base resource will request version 1.0 of the API if available.
-* The versioning format is v&lt;#MAJOR&gt;.&lt;#MINOR&gt;
-* MINOR increments SHOULD be performed for non-breaking changes (such as the addition of attributes in a response)
-* MAJOR increments MUST be performed for breaking changes (such as the renaming of a resource or attribute)
-* Versions MUST be represented as complete strings. Parsing MUST proceed as follows: separate into two strings, using the point (.) as a delimiter. Compare integer representations of MAJOR, MINOR version (such that v1.12 is greater than v1.5).
-* Clients are responsible for identifying the correct API version they require.
+- Appending /v1.0/ to the API base resource will request version 1.0 of the API if available.
+- The versioning format is v&lt;#MAJOR&gt;.&lt;#MINOR&gt;
+- MINOR increments SHOULD be performed for non-breaking changes (such as the addition of attributes in a response)
+- MAJOR increments MUST be performed for breaking changes (such as the renaming of a resource or attribute)
```

Comment:
These requirements are on specification maintainers not implementers, so as we've discussed previously, these could be rephrased as:

```suggestion
- MINOR increments will be performed for non-breaking changes (such as the addition of attributes in a response)
- MAJOR increments will be performed for breaking changes (such as the renaming of a resource or attribute)
```

This language is also repeated in other specs...